src/data/consul.json

```diff
@@ -28,5 +29,32 @@
         "href": "https://support.hashicorp.com"
       }
     ]
-  }
+  },
+  "navigationHeaderItems": [
```

Comment:
This is likely not final, fully correct `navigationHeaderItems` - I based this off the current `.io` nav. The intent here is to have something to navigate to the new `docs` routes with.